library/src/fy_library/mixins/property/property_setter_imports/using_property_mixin_fy.py

```diff
@@ -0,0 +1,38 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+#  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+"""fy
+from typing import List
+from fy_library.domain.fy_py_template_models import PropertyMixinModel
+
+
+property property_setter_imports: List[PropertyMixinModel] using property_mixins:
```

Comment:
Rename
```suggestion
property property_setter_mixins: List[PropertyMixinModel] using property_mixins:
```

These are mixins not imports.